Lib/ufo2ft/featureWriters/kernFeatureWriter.py

```diff
@@ -198,6 +198,17 @@ def setContext(self, font, feaFile, compiler=None):
         ctx.gdefClasses = self.getGDEFGlyphClasses()
         ctx.glyphSet = self.getOrderedGlyphSet()
 
+        # Remember which languages are defined for which OT tag, as all
+        # generated kerning needs to be registered for the script's `dflt`
+        # language, but also all those the designer defined manually. Otherwise,
+        # setting any language for a script would deactivate kerning.
+        feaScripts = ast.getScriptLanguageSystems(feaFile, excludeDflt=False)
```

Comment:
nit: I would call this feaLanguages or feaLangsByScript -- the scripts are the key, but the values are lists of languages, the values is the important bit usually